### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by kubraa16**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -3,12 +3,35 @@ import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
...
+import CustomTab from "./components/core/products/CustomTab.jsx";
+import ProductsList from "./components/core/products/ProductsList.jsx";
 function App() {
   return (
     <Router>
       <Routes>
-        <Route path="/" element={<IndexTable />} />
-        <Route path="/company/:ticker" element={<CompanyDetails />} />
+        {/* For assignemnt 2 and 3  */}
+        {/* <Route path="/" element={<IndexTable />} /> */}
```
Comment (gauravvolopay) [inline src/App.jsx]: Why are these routes commented ?

File: src/App.jsx [hunk 2]
```diff
@@ -3,12 +3,35 @@ import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
...
+        {/* <Route path="/company/:ticker" element={<CompanyDetails />} /> */}
+
+        {/*  For assignment 4  */}
+        <Route
+          path="/"
```
Comment (gauravvolopay) [inline src/App.jsx]: Route for Product component is **/products**

**Review by gauravvolopay — CHANGES_REQUESTED** [review]

File: src/components/core/products/CustomTab.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import React, { Component, useEffect, useState } from "react";
```
Comment (gauravvolopay) [inline src/components/core/products/CustomTab.jsx]: Remove products folder from **core** folder, as core components are dumb components and these are not dumb components

**Review by gauravvolopay — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (kubraa16) [reply src/App.jsx]: created a Navbar to navigate between stocks and products overview.

File: src/App.jsx (hunk 2 above)
Comment (kubraa16) [reply src/App.jsx]: done.

File: src/components/core/products/CustomTab.jsx (hunk 3 above)
Comment (kubraa16) [reply src/components/core/products/CustomTab.jsx]: Done .

**Review by gauravvolopay — APPROVED** [review]

**Pull request merged**